sources/osgAnimation/AnimationAttribute.js

```diff
@@ -11,28 +11,34 @@ define( [
      * @class AnimationAttribute
      * @inherits StateAttribute
      */
-    var AnimationAttribute = function ( disable ) {
+    var AnimationAttribute = function ( boneSize, animationID, disable ) {
         StateAttribute.call( this );
+        this._boneSize = boneSize;
         this._enable = !disable;
+
+        this._animationID = animationID;
+        if ( !animationID ) AnimationAttribute.getAnimationID();
```

Comment:
mmm was afraid of unexpected side effect if two object share the same instanceID 
(clone & original would have the same )